User_Service/Controllers/controller.go

```diff
@@ -0,0 +1,270 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	data "github.com/E-Furqan/Food-Delivery-System/Data"
+	config "github.com/E-Furqan/Food-Delivery-System/database_config"
+	"github.com/E-Furqan/Food-Delivery-System/utils"
+	"github.com/gin-gonic/gin"
+)
+
+func Register(c *gin.Context) {
```

Comment:
we should follow CLEAN architecture

Reply:
have followed the clean architecture and made the changes from data to entity and moved controllers into interfaces folder and have made a separate package for db functions